### PR TITLE
fix #1453 Feature/cv2 

### LINKED ISSUE #1453: Refactor items wrapping in ActionBar

### PR DESCRIPTION
**Pull request by OlgaLarina**

### CODE REVIEW

File: packages/survey-creator-core/src/components/action-container-view-model.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          needSeparator: items.length > 0,
+          title: this.creator.getLocString("pe.delete"),
+          visibleIndex: 30,
+          action: () => {
+            this.creator.deleteElement(element);
+          }
+        })
+      );
+    }
+    return items;
```
Comment (tsv2013) [inline packages/survey-creator-core/src/components/action-container-view-model.ts:51]: We need to raise the  from creator here:
```JS
    this.onDefineElementMenuItems.fire(this, {
      obj: element,
      items: items
    });

```

File: packages/survey-creator-core/src/components/tabs/logic-ui.ts [hunk 2]
```diff
@@ -67,8 +69,13 @@ export class SurveyLogicUI extends SurveyLogic {
    * The list of toolbar items. You may add/remove/replace them.
    * @see IActionBarItem
```
Comment (tsv2013) [inline packages/survey-creator-core/src/components/tabs/logic-ui.ts:70]: IAction, we need to find all referencies

File: packages/survey-creator-core/src/creator-base.ts [hunk 3]
```diff
@@ -112,7 +114,7 @@ export class CreatorBase<T extends SurveyModel>
   @property({ defaultValue: "" }) currentAddQuestionType: string;
   private isRTLValue: boolean = false;
   private alwaySaveTextInPropertyEditorsValue: boolean = false;
-  private toolbarItemsValue: CreatorToolbarItems;
+  private toolbarItemsValue: AdaptiveActionContainer;
```
Comment (tsv2013) [inline packages/survey-creator-core/src/creator-base.ts:117]: Do we need to rename the toolbarItemsValue?

File: packages/survey-creator-core/src/components/action-container-view-model.ts (hunk 1 above)
Comment (OlgaLarina) [reply packages/survey-creator-core/src/components/action-container-view-model.ts:51]: This event is fired in onElementMenuItemsChanged method

**Review by tsv2013 — APPROVED** [review]

**Pull request merged**